app/assets/stylesheets/opportunities.scss

```diff
@@ -0,0 +1,45 @@
+@import "foundation_and_overrides";
+
+.vol-panel, .opportunity-info {
+  border: 5px solid #FF851B;
+  color: white;
+  background-color: #496d7b;
```

Comment:
Color literals like `#496d7b` should only be used in variable declarations; they should be referred to via variable everywhere else.